src/index.tsx

```diff
@@ -136,13 +136,16 @@ const addScriptTag = (loadCb: Function, url?: string) => {
   return tag
 }
 
-const PinwheelModal = ({
-  open,
-  _srcUrl,
-  // @ts-ignore
-  _modalSessionIdOverride,
-  ...props
-}: PinwheelModalProps) => {
+const PinwheelModal = (allProps: PinwheelModalProps) => {
+  const { open, _srcUrl, ...props } = allProps
+
+  // Need to get _modalSessionIdOverride like this or else client using this module
+  // with strict typescript typing will not be able to compile this without an error.
+  // Error: "node_modules/@pinwheel/react-modal/dist/index.d.ts(85,48): error TS2339: Property '_modalSessionIdOverride' does not exist on type 'PinwheelModalProps'."
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const propsAsAny = allProps as unknown as any
```

Comment:
Why not just add this type to `PinwheelModalProps`?

Reply:
It's not a part of our public contract!